examples/prometheus-alerting-rules/alerts.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+- name: kube-state-metrics
+  rules:
+  - alert: KubeStateMetricsListErrors
+    annotations:
+      message: kube-state-metrics is experiencing list errors
```

Comment:
maybe "kube-state-metrics is experiencing elevated errors in list operations" ? (same for the other just "list" replaced with "watch")